kiali/README.md

```diff
@@ -67,16 +83,19 @@ For integration instructions, read [Kiali: Jaeger configuration](https://kiali.i
 
 ### Integrate Grafana
 
-Kiali can provide links to Istio dashboards in Grafana. 
+Kiali can provide links to Istio dashboards in Grafana.
 
 For integration instructions, read [Kiali: Grafana configuration](https://kiali.io/docs/configuration/p8s-jaeger-grafana/grafana/).
 
-### Expose Kiali
+### Authentication
 
-Kiali supports different authentication strategies. When exposing the Kiali server, we recommend to use an external identity provider compatible with OpenID Connect (OIDC).
+Kiali supports different authentication strategies. The default authentication strategy uses a Kubernetes Service Account Token. If you use a kubeconfig file with a static token, you can use this token to authenticate. Depending on your preferred way to access Kiali, different authentication strategies might be suitable. To learn more about Kiali authentication strategies, read [Kiali: Authentication Strategies](https://kiali.io/docs/configuration/authentication/).
 
-1. Set up Kiali with the [OpenID Connect strategy](https://kiali.io/docs/configuration/authentication/openid/).
-1. To expose Kiali, follow the [Expose a workload](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway/) instructions.
+* For Kiali access by port forwarding, you need no additional authentication, and you can activate the [anonymous strategy](https://kiali.io/docs/configuration/authentication/anonymous/):
+  ```bash
+  helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
```

Comment:
Pardon me, I forgot that it works with or without the curly brackets when I added the comment. It must have been because of the other error I got that I just connected the wrong dots. 
But either way, without the brackets is the convention we use in Kyma documentation for environment variables, and we use the brackets for placeholders, so it's still good that you removed them. I just wanted to correct myself :)